external/protobuf/CMakeLists.txt

```diff
@@ -48,5 +48,19 @@ if (EMIL_HOST_BUILD)
         endforeach()
     endfunction()
 
+    function(remove_mp)
+        foreach(target ${ARGN})
+            get_target_property(all_options ${target} COMPILE_OPTIONS)
+            if(all_options)
+                list(REMOVE_ITEM all_options "/MP")
+                set_target_properties(${target} PROPERTIES COMPILE_OPTIONS "${all_options}")
+            endif()
+        endforeach()
+    endfunction()
+
     add_protobuf_target_properties(libprotobuf libprotoc)
+
+    # /MP is added by protobuf in case of MSVC, but that check is a (partial) false positive since we use the MSVC _frontend_ for clang
+    # which does not support this flag
+    remove_mp(libprotobuf libprotobuf-lite libprotoc libupb)
```

Comment:
But there is also a configuration with plain old MSVC, so /MP should not be removed unconditionally